From 279458dc3f2a3a2d40c0b6316d187db3133606d8 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 14 Jun 2016 16:32:46 +0000
Subject: [PATCH 0764/1076] MANTIS 0040313 platform.idl V3.2 added token for
 MediaOffRolling allow CLA to access non media off rolling devices, while
 platform uses media off rolling devices.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1038 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index d4b667b2..a118e23b 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "3.1";
+    const string interface_version = "3.2";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -49,6 +49,8 @@ module PlatformManagement
         TOKEN_APP_ACTIVE, /**< token of the currently ACTIVE application */
         TOKEN_APP_INIT,   /**< token of the currently INITILAIZE application */
         TOKEN_APP_IDLE,   /**< token of a currently AVAILABLE or UNAVAILABLE application */
+        TOKEN_MOR,        /**< token of media off rolling module */
+        TOKEN_MOR_ACTIVE, /**< token of media off rolling module, that is currently active */
         TOKEN_INVALID     /**< invalid token */
     };
 
@@ -139,7 +141,7 @@ module PlatformManagement
         /** prepare a device for using by CLA in media-roll-off mode. 
             record events for playback after next prepareDevice
         */
-        code beginMediaRecord(in string contextName);
+        code beginMediaRecord(in string contextName, in string token);
         
         /** same as resetDevices; except for mediaOffRoller components
             the mediaOffRoller components will playback the stored events
-- 
2.41.0.windows.1

